dandiapi/api/tests/test_dandiset.py

```diff
@@ -571,7 +571,7 @@ def test_dandiset_rest_create_with_contributor(api_client, admin_user):
             'created': TIMESTAMP_RE,
             'modified': TIMESTAMP_RE,
         },
-        'contact_person': 'Jane Doe',
+        'contact_person': 'Doe, John',
```

Comment:
This is a subtle behavior change-- Jane isnt "the" contact person anymore, shes 1 of 2, and John shows up first. @yarikoptic should there only be 1 contact person?